src/pal/src/exception/seh-unwind.cpp

```diff
@@ -107,12 +107,16 @@ static void UnwindContextToWinContext(unw_cursor_t *cursor, CONTEXT *winContext)
 
 static void GetContextPointer(unw_cursor_t *cursor, int reg, PDWORD64 *contextPointer)
 {
+#if defined(__APPLE__)
+    unw_get_reg(cursor, reg, (unw_word_t *) contextPointer);
```

Comment:
Yeah, I was aware it wasn't correct, but there is no save_loc analogue that I could find last night.